<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
    background-color: black
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>551</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: black; /* تنظیم رنگ و ضخامت خط دور */
    
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: white;
}</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userNamelineEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border-radius: 10px; /* مقدار این عدد را به دلخواه تغییر دهید */
    border: 1px solid black; /* تنظیم رنگ و ضخامت خط دور */
    padding: 5px; /* تنظیم فاصله متن از لبه‌ها */
}
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: white;
}</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordlineEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border-radius: 10px; /* مقدار این عدد را به دلخواه تغییر دهید */
    border: 1px solid black; /* تنظیم رنگ و ضخامت خط دور */
    padding: 5px; /* تنظیم فاصله متن از لبه‌ها */
}
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="login_pushbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px solid black; /* تنظیم رنگ و ضخامت خط دور */
    border-radius: 10px; /* مقدار این عدد را به دلخواه تغییر دهید تا گردی گوشه‌ها را تنظیم کنید */
    background-color: #887f94; /* تنظیم رنگ پس‌زمینه */
    color: white; /* تنظیم رنگ فونت */
    padding: 5px; /* تنظیم فاصله متن از لبه‌ها */
}
</string>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>660</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>217</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #8e8398;
}</string>
      </property>
      <property name="text">
       <string>Don't have an account?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signUpButton">
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    color: #00f;
    background-color: transparent;
    border: none;
    text-decoration: underline;
    cursor: pointer;
}
</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
